### PR TITLE
adds usage description

### PR DESCRIPTION
**Pull request by TeresaNesteby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,26 @@ it, simply add the following line to your `Podfile`:
 pod "Segment-Appboy"
 ```
 
+## Usage
+
+After adding the dependency, you must register the integration with our SDK. To do this, import the Appboy integration in your AppDelegate:
+
+
+```
+#import <Segment-Appboy/AppboyIntegrationFactory.h>
```
Comment (chshapiro) [inline README.md]: I tested with the import statement that you have here, and the project didn't build. Would you be able to either replace it with the import statement we use in our sample app (`#import "SEGAppboyIntegrationFactory.h"`) or share your Podfile and integration method so we can see how you're integrating?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,26 @@ it, simply add the following line to your `Podfile`:
...
+And add the following lines:
+
+```
+NSString *const SEGMENT_WRITE_KEY = @" ... ";
+SEGAnalyticsConfiguration *config = [SEGAnalyticsConfiguration configurationWithWriteKey:SEGMENT_WRITE_KEY];
+
+[config use:[SEGAppboyIntegrationFactory instance]];
+
+[SEGAnalytics setupWithConfiguration:config];
+```
```
Comment (chshapiro) [inline README.md]: Can you just add this line with our documentation link at the end?
`Please see our documentation for more info: https://segment.com/docs/integrations/appboy/#ios`

File: README.md (hunk 1 above)
Comment (TeresaNesteby) [inline README.md]: @chshapiro Great catch. Updated!

**Comment by chshapiro** [comment]
Thank you for the quick fixes! I will merge now.

**Pull request merged**